airflow/providers/amazon/aws/waiters/emr-containers.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": 2,
+    "waiters": {
+        "container_job_complete": {
+            "operation": "DescribeJobRun",
+            "delay": 30,
+            "maxAttempts": 60,
+            "acceptors": [
+                {
+                    "matcher": "path",
+                    "argument": "jobRun.state",
+                    "expected": "COMPLETED",
+                    "state": "success"
+                }
```

Comment:
Can we also include here another acceptor for the terminal FAILED state

ref: https://docs.aws.amazon.com/batch/latest/APIReference/API_DescribeJobs.html#API_DescribeJobs_ResponseSyntax
https://docs.aws.amazon.com/batch/latest/userguide/job_states.html

Otherwise, the waiter will have to wait until max_attempts are done although the Batch job might have failed already.

Reply:
+1 to this